src/V3EmitCImp.cpp

```diff
@@ -343,6 +342,48 @@ class EmitCImp final : EmitCFunc {
             puts("}\n");
             splitSizeInc(10);
         }
+        if (v3Global.opt.coverageToggle()) {
+            puts("\n// Toggle Coverage\n");
+            puts("void " + prefixNameProtect(m_modp) + "::__vlCoverToggleInsert(");
+            puts("int begin, int end, bool ranged, ");
+            puts(v3Global.opt.threads() > 1 ? "std::atomic<uint32_t>" : "uint32_t");
+            puts("* countp, bool enable, const char* filenamep, int lineno, int column,\n");
+            puts("const char* hierp, const char* pagep, const char* commentp) {\n");
+            if (v3Global.opt.threads() > 1) {
+                puts("assert(sizeof(uint32_t) == sizeof(std::atomic<uint32_t>));\n");
+            }
+            puts("int step = (end >= begin) ? 1 : -1;\n");
+            // range is inclusive
+            puts("for (int i = begin; i != end + step; i += step) {\n");
+            if (v3Global.opt.threads() > 1) {
+                puts("uint32_t* count32p = reinterpret_cast<uint32_t*>(countp);\n");
+            } else {
+                puts("uint32_t* count32p = countp;\n");
```

Comment:
```suggestion
                puts("uint32_t* const count32p = reinterpret_cast<uint32_t*>(countp);\n");
            } else {
                puts("uint32_t* const count32p = countp;\n");
```

Reply:
I had to revert it, because there is an assignment to that pointer below:
https://github.com/verilator/verilator/blob/5a6b49cdde3d7a3b733256fc07e718046036c142/src/V3EmitCImp.cpp#L331